lib/travis/model/repository.rb

```diff
@@ -43,7 +43,7 @@ def with_builds
       where(arel_table[:last_build_id].not_eq(nil))
     end
 
-    def administratable
+    def administrable
```

Comment:
This is probably an api change. I can split it/drop it/add any necessary aliasing as requested/required.